### PR TITLE
feat: Add missing properties to SourcePostModeration

### PR DESCRIPTION
**Pull request by AmarTrebinjac**
Added props needed for edit flow

### CODE REVIEW

File: src/schema/posts.ts [hunk 1]
```diff
@@ -115,6 +116,8 @@ export interface GQLSourcePostModeration {
   status: SourcePostModerationStatus;
   createdAt: Date;
   updatedAt: Date;
+  source: Source;
```
Comment (capJavert) [inline src/schema/posts.ts:119]: source would also be optional?

**Review by capJavert — APPROVED** [review]

**Comment by sshanzel** [comment]
This should be added here too: https://github.com/dailydotdev/daily-api/pull/2443

**Review by sshanzel — APPROVED** [review]

File: src/schema/posts.ts (hunk 1 above)
Comment (sshanzel) [reply src/schema/posts.ts]: Source is always there compared to post, if I'm not mistaken.

File: src/schema/posts.ts [hunk 2]
```diff
@@ -276,6 +279,10 @@ export const typeDefs = /* GraphQL */ `
     """
     sourceId: String
     """
+    Source
+    """
+    source: Source
+    """
```
Comment (capJavert) [inline src/schema/posts.ts:285]: Then here it should be
```suggestion
    Source
    """
    source: Source!
    """
```

File: src/schema/posts.ts (hunk 1 above)
Comment (capJavert) [reply src/schema/posts.ts]: You are right, then in schema it should be https://github.com/dailydotdev/daily-api/pull/2449#discussion_r1842118045

Comment (capJavert) [reply src/schema/posts.ts]: @AmarTrebinjac also you can revert https://github.com/dailydotdev/daily-api/pull/2449/commits/21c0e6ff8dad063bca27dc704a414b58ee3571ef, sorry for the juggle 😆

Comment (AmarTrebinjac) [reply src/schema/posts.ts]: @AmarTrebinjac haha, yep. I did a double take and checked, but my local branch is memeing now 💩💩

**Pull request merged**